openmc/material.py

```diff
@@ -403,6 +403,32 @@ def add_nuclide(self, nuclide: str, percent: float, percent_type: str = 'ao'):
 
         self._nuclides.append(NuclideTuple(nuclide, percent, percent_type))
 
+    def add_nuclides(self, nuclides: dict):
+        """ Add multiple nuclides to a material
+
+        Parameters
+        ----------
+        nuclides : dict of str to tuple
+            Dictionary mapping nuclide names to a tuple containing their
+            atom or weight percent.
+
+        """
+
+        for nuclide, (percent, percent_type) in nuclides.items():
+                self.add_nuclide(nuclide, percent, percent_type)
+
+    def add_elements(self, elements: dict):
+        """ Add multiple elements to a material
+
+        Parameters
+        ----------
+        elements : dict of str to tuple
+            Dictionary mapping element names to a tuple containing their
+            atom or weight percent.
+        """
+
+        for element, (percent, percent_type) in elements.items():
+                self.add_element(element, percent, percent_type)
     def remove_nuclide(self, nuclide: str):
```

Comment:
Sorry I missed the line break last time

```suggestion
                self.add_element(element, percent, percent_type)

    def remove_nuclide(self, nuclide: str):
```